src/views/account/Chat/Messages.tsx

```diff
@@ -61,7 +61,7 @@ const Discussions: Screen<"Discussions"> = ({ navigation, route }) => {
   const [chats, setChats] = useState<Chat[] | null>(null);
   const [refreshing, setRefreshing] = useState<boolean>(false);
 
-  const supported = account.service === AccountService.Pronote;
+  const supported = account.service === AccountService.Pronote || AccountService.EcoleDirecte;

```

Comment:
J'pense pas que ça marche, j'aurai plutôt fait ça :

```suggestion
  const supported = account.service === AccountService.Pronote || account.service === AccountService.EcoleDirecte;
```